readme.md

```diff
@@ -0,0 +1,41 @@
+# Ruby CLI Lab
+
+Today is _your_ chance to practice Ruby. You'll have the option to choose one of
+three prompts:
+
+- [Battleship](./battleship.md)
+- [Flash Cards](./flash_cards.md)
+- [Personal Finance](./personal_finance.md)
```

Comment:
Battleship is def the hardest of the three. Should let students know, either in the readme or IRL during the intro.